static/js/jira-server-url.js

```diff
@@ -22,15 +22,11 @@ const checkValidGHEUrl = inputURL => {
 	try {
 		const { protocol, hostname } = new URL(inputURL);
 
-		if (!ALLOWED_PROTOCOLS.includes(protocol)) {
-			setErrorMessage(defaultError);
-			return false;
-		}
-		// This checks whether the hostname whether there is an extension like `.com`, `.net` etc.
-		if (hostname.split('.').length < 2) {
```

Comment:
Removed this entirely because it doesn't actually validate if there is a TLD.

Uploading Screen Recording 2022-07-04 at 2.06.56 pm.mov…